profiling/README.md

```
@@ -0,0 +1,69 @@
+
+## Reproduction
+
+### Prerequisites
+- Ubuntu 22.04 (tested on x86_64, 32GB RAM)
+- AD4M executor binary (v0.11.1 or from this branch)
+- Node.js 18+
+- Bootstrap languages published or available as seed
+
+### Running the Leak Investigation
+
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)